lib/workers/global/limits.ts

```diff
@@ -37,3 +38,159 @@ export function isLimitReached(key: Limit): boolean {
   const { max, current } = limit;
   return max - current <= 0;
 }
+
+export type CountName = 'ConcurrentPRs' | 'HourlyPRs' | 'Branches';
+
+type BranchLimitName =
+  | 'branchConcurrentLimit'
+  | 'prConcurrentLimit'
+  | 'prHourlyLimit';
+
+export const counts = new Map<CountName, number>();
+
+export function setCount(key: CountName, val: number): void {
+  const count = val;
+  counts.set(key, count);
+  logger.debug(`${key} count = ${count}`);
+}
+
+export function incCountValue(key: CountName, incBy = 1): void {
+  const count = counts.get(key) ?? 0;
+  counts.set(key, count + incBy);
+}
+
+function handleOtherLimits(
```

Comment:
```suggestion
function handleConcurrentLimits(
```
IMO this is a more fitting naming